pyproject.toml

```diff
@@ -127,9 +127,3 @@ addopts = [
     "--block-network",
     "--allowed-hosts=127.0.0.1,::1",
 ]
-python_files = [
-    "tests.py",
-    "test_*.py",
```

Comment:
how come we're losing the test_*.py above signatures ?  don't we need this for the new location/files ?

Reply:
not anymore =D

Reply:
pytest defaults covers what we have now. ty @infinitewarp <3